boards/same54-xpro/board.c

```diff
@@ -20,6 +20,38 @@
 
 #include "board.h"
 #include "periph/gpio.h"
+#include "mtd_spi_nor.h"
+#include "timex.h"
+
+#ifdef MODULE_MTD
+/* N25Q256A */
+static const mtd_spi_nor_params_t _same54_nor_params = {
+    .opcode = &mtd_spi_nor_opcode_default,
+    .wait_chip_erase = 240 * US_PER_SEC,
+    .wait_32k_erase = 700 * US_PER_MS,
+    .wait_sector_erase = 250 * US_PER_MS,
+    .wait_chip_wake_up = 1 * US_PER_MS,
```

Comment:
Just wondering,
Where do these values come from ?

Reply:
Got them from the [data sheet](https://www.micron.com/-/media/client/global/documents/products/data-sheet/nor-flash/serial-nor/n25q/n25q_256mb_3v.pdf)

![image](https://user-images.githubusercontent.com/1301112/100132335-93cac680-2e85-11eb-90e3-5360a047c7b8.png)